src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoQueryModelDefinition.xml

```diff
@@ -29,7 +29,7 @@
   
   <Node Name="SqlEndpoint" Parent="Server"/>
   <Node Name="SqlLinkedServer" Parent="Server" />
-  <Node Name="SqlServerDdlTrigger" Parent="Server" >
+  <Node Name="SqlServerDdlTrigger" Parent="Server" ValidFor="Sql2005|Sql2008|Sql2012|Sql2014|Sql2016|Sql2017|AzureV12" >
```

Comment:
it seems to be: [https://docs.microsoft.com/en-us/sql/t-sql/statements/create-trigger-transact-sql](https://docs.microsoft.com/en-us/sql/t-sql/statements/create-trigger-transact-sql)

---
In reply to: [122100917](https://github.com/Microsoft/sqltoolsservice/pull/380#discussion_r122100917) [](ancestors = 122100917)